fix: Discord breaking changes

naomi-lgbt opened the pull request:
**Description:**

Discord rolled out breaking changes to the channel creation endpoint.
Our private command was failing due to these changes - by assigning
access to the bot user instead of the bot role, this issue appears to
be resolved.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>
<!--Is this related to an issue? Does it close one? If so, replace the XXXXX below with the issue number.-->

Closes #XXXXX

Commit:
fix: Discord breaking changes

Discord rolled out breaking changes to the channel creation endpoint.
Our private command was failing due to these changes - by assigning
access to the bot user instead of the bot role, this issue appears to
be resolved.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

bradtaniguchi reviewed (changes requested):
As going thru this feature I noticed that if you just call `FCC moderate` you get the following:
```
undefined is not a valid action for this command. Please try again, and tell me if you would like to suspend, private, or close.
```

which is kinda weird for a few reasons. 1. `undefined` is always weird. 2. `suspend` is no longer supported.

---
If forgot to set the `PRIVATE_CATEGORY` environment variable for my test server and the bot thru an error in the console and didn't report anything to the end user in discord:
```
(node:1993) UnhandledPromiseRejectionWarning: DiscordAPIError: Invalid Form Body
name: This field is required
    at RequestHandler.execute (/home/codespace/workspace/discord-bot-test/node_modules/discord.js/src/rest/RequestHandler.js:170:25)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1993) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1993) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
(I think it crashed it?)

---

Other wise it worked great for the `suspend` and `close` flows :)

Commit:
fix: apply review feedback

- Adds a fallback value for missing `PRIVATE_CATEGORY` env
- Adds separate check + message for missing `action` value in `moderate`
command.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

bradtaniguchi reviewed (approved):
LGTM :+1:
